src/api/config/corsConfig.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * Purpose: CORS Configuration for Express API
+ */
```

Comment:
This is fine for this PR, but look up JSDoc comment syntax. It's built-in and is already a standard.

Reply:
@Bottle7 PO said to add author tags, so I'll update this as well. Forgot the summary tag existed 🤔